common/utils/src/main/resources/error/error-classes.json

```diff
@@ -6705,6 +6705,259 @@
       "Failed to get block <blockId>, which is not a shuffle block"
     ]
   },
+  "_LEGACY_ERROR_TEMP_3050" : {
+    "message" : [
+      "Cannot modify the value of a static config: <k>"
+    ]
```

Comment:
Can we use this?
- https://spark.apache.org/docs/latest/sql-error-conditions.html#cannot_modify_config